src/lib/evm/on-chain/evm-transfer-subscriptions/evm-http-rpc-listener.ts

```diff
@@ -60,13 +61,14 @@ export abstract class EvmHttpRpcListener<T extends unknown[] = []> extends Liste
     }
   }
 
-  protected async onError(error: unknown) {
-    if (!this.isActive) {
-      return;
-    }
+  protected async onError(error: any) {
+    if (!this.isActive) return;
 
-    console.error(error);
     this.stopListening();
+
+    if (error.details === FILTER_NOT_FOUND_ERROR) return;
```

Comment:
If you made sure that the events are listened even after this error, just resolve this conversation